scs2-session-visualizer-jfx/src/main/java/us/ihmc/scs2/sessionVisualizer/jfx/session/log/MultiVideoDataReader.java

```diff
@@ -27,8 +27,23 @@ public MultiVideoDataReader(File dataDirectory, LogProperties logProperties, Bac
          Camera camera = cameras.get(i);
          try
          {
-            VideoDataReader reader = new VideoDataReader(camera, dataDirectory, logProperties.getVideo().getHasTimebase());
-            readers.add(reader);
+            VideoDataReader reader;
+            if (camera.getType().toString().equals("MAGEWELL_CAPTURE"))
```

Comment:
Depending on the type of capture algorithm used, we can create a differ VideoDataReader to load it correctly

Reply:
This seems kind of fragile, why would you not compare against the enum CameraType?